ouroboros-network-api/src/Ouroboros/Network/AnchoredSeq.hs

```diff
@@ -295,6 +296,26 @@ splitAt i (AnchoredSeq a ft) = case FT.split (\v -> measureSize v > i) ft of
      let before' = AnchoredSeq a before
      in (before', AnchoredSeq (headAnchor before') after)
 
+-- | \( O(\log(\min(i,n-i)) \). Split the 'AnchoredSeq' at a given
+--  measure.
+--
+-- POSTCONDITION: @(before, after) = splitAtMeasure v s@, then:
+--
+-- > anchor      before == anchor s
+-- > headAnchor  before == anchor after
+-- > headAnchor  after  == headAnchor s
+-- > [before] == filter ((< v) . getElementMeasure . MeasuredWith) s
+-- > [after] == filter ((v <=) . getElementMeasure . MeasuredWith) s
+splitAtMeasure ::
```

Comment:
What about having a `AnchoredFragment`-specific wrapper (eg `splitAtSlot`), just like `splitAfterPoint` is a wrapper around `splitAfterMeasure`?

Reply:
I added `splitAtSlot`.